Resources/Prototypes/_Harmony/Catalog/Fills/Crates/armory.yml

```
@@ -44,3 +44,27 @@
       - id: SpeedLoaderLightRifle
         orGroup: ammo
         amount: 42
+
+- type: entity
+  id: CrateArmorySMGHarmony
+  suffix: Drozd
+  parent: [ CrateWeaponSecure, BaseSecurityContraband ]
+  name: SMG crate
+  description: Contains two high-powered, automatic rifles with four mags. Requires Armory access to open.
+  components:
+  - type: StorageFill
+    contents:
+      - id: WeaponSubMachineGunDrozd
+        amount: 2
+
+- type: entity
+  id: CrateArmoryShotgunHarmony
+  suffix: Kammerer
```

The crate should also keep the secure suffix
```suggestion
  suffix: Kammerer, Secure
```